client/my-sites/domains/domain-management/transfer/transfer-page/index.tsx

```diff
@@ -1,20 +1,59 @@
+import { Card } from '@automattic/components';
+import { useI18n } from '@wordpress/react-i18n';
 import { connect } from 'react-redux';
+import ActionCard from 'calypso/components/action-card';
 import Main from 'calypso/components/main';
 import BodySectionCssClass from 'calypso/layout/body-section-css-class';
 import { getSelectedDomain, isMappedDomain } from 'calypso/lib/domains';
+import {
+	domainManagementTransferToAnotherUser,
+	domainManagementTransferToOtherSite,
+} from 'calypso/my-sites/domains/paths';
 import getCurrentRoute from 'calypso/state/selectors/get-current-route';
 import getPrimaryDomainBySiteId from 'calypso/state/selectors/get-primary-domain-by-site-id';
 import isDomainOnlySite from 'calypso/state/selectors/is-domain-only-site';
 import isPrimaryDomainBySiteId from 'calypso/state/selectors/is-primary-domain-by-site-id';
 import isSiteAutomatedTransfer from 'calypso/state/selectors/is-site-automated-transfer';
 import { hasLoadedSiteDomains } from 'calypso/state/sites/domains/selectors';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import type { TransferPageProps } from './types';
+
+import './style.scss';
+
+const TransferPage = ( props: TransferPageProps ): JSX.Element => {
+	const { __ } = useI18n();
+	const { selectedSite, selectedDomainName, currentRoute } = props;
 
-const TransferPage = (): JSX.Element => {
 	return (
-		<Main className="transfer-page" wideLayout>
+		<Main wideLayout className="transfer-page">
 			<BodySectionCssClass bodyClass={ [ 'edit__body-white' ] } />
-			The page goes here
+			<Card>
+				<ActionCard
+					buttonHref={ domainManagementTransferToAnotherUser(
+						selectedSite.slug,
+						selectedDomainName,
+						currentRoute
+					) }
+					// translators: Continue is a verb
+					buttonText={ __( 'Continue' ) }
+					// translators: Transfer a domain to another user
+					headerText={ __( 'To another user' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **27** times:
`translate( 'Transfer to another user' )` *ES Score: 6*